.gitignore

```diff
@@ -12,5 +12,14 @@ build/
 # OSX dir files
 .DS_Store
 
+# Dep
+_build/
+double-conversion/
```

Comment:
hmm... we should also fix the script so they don't pollute root level ? lol

Reply:
We can use this `DEPENDENCY_DIR` , defined in https://github.com/facebookresearch/torcharrow/blob/e03beb1750e13f96f8b8a049b6b82fb38c927841/scripts/_setup-macos.sh#L38

Here is an example usage in Velox CI: 
https://github.com/facebookincubator/velox/blob/776f9840462f7d2b5053740ffd61c5202eeeee67/.circleci/config.yml#L69

Reply:
I see. I can move them into `_build` directory.

Reply:
to align with the behavior on ubuntu

Reply:
@ejguan : Feel free to do it in a separate PR